tests/components/traccar/test_init.py

```diff
@@ -0,0 +1,230 @@
+"""The tests the for Traccar device tracker platform."""
+from unittest.mock import Mock, patch
+
+import pytest
+
+from homeassistant import data_entry_flow
+from homeassistant.components import traccar, zone
+from homeassistant.components.device_tracker import (
+    DOMAIN as DEVICE_TRACKER_DOMAIN)
+from homeassistant.components.traccar import DOMAIN, TRACKER_UPDATE
+from homeassistant.const import (
+    HTTP_OK, HTTP_UNPROCESSABLE_ENTITY, STATE_HOME, STATE_NOT_HOME)
+from homeassistant.helpers.dispatcher import DATA_DISPATCHER
+from homeassistant.setup import async_setup_component
+
+HOME_LATITUDE = 37.239622
+HOME_LONGITUDE = -115.815811
+
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture(autouse=True)
+def mock_dev_track(mock_device_tracker_conf):
+    """Mock device tracker config loading."""
+    pass
+
+
+@pytest.fixture
+async def traccar_client(loop, hass, aiohttp_client):
+    """Mock client for Traccar (unauthenticated)."""
+    assert await async_setup_component(
+        hass, 'persistent_notification', {})
+
+    assert await async_setup_component(
```

Comment:
@MartinHjelmare This seems to setup the device_tracker traccar component, not the new config_flow traccar component. Can you help me here?

Reply:
I see you found it. :+1:

I was going to ref this:
https://github.com/home-assistant/home-assistant/blob/8f2f77083761cd58d186dd7ee6a23e8b4be3cf08/tests/components/gpslogger/test_init.py#L62-L75